changelog.d/620.change.rst

```diff
@@ -0,0 +1,5 @@
+Fixed serialization behavior of non-slots classes with ``cache_hash=True``. Operations which make "deep copies" of instances of classes with hash caching, the hash cache will be cleared on copy (though the cache will not be cleared with shallow copies like those made by ``copy.copy``).
```

Comment:
Sorry I hate it but I have to be a bit nitpicky here. :(

- Firstly: we probably should delete `changelog.d/611.change.rst`?
- 1 sentence per line please.
- functions like `copy.copy` please write with () i.e. `copy.copy()`